pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -61,8 +61,9 @@ stdenv.mkDerivation rec {
       cat > $out/bin/liquibase <<EOF
       #!/usr/bin/env bash
       # taken from the executable script in the source
-      CP="$out/liquibase.jar"
+      CP="$out/liquibase-core.jar"
```

Comment:
can probably start with `$CP=""`, since the `addJars` command below will add `postgresql.jar`, `liquibase-core.jar`, and `picocli.jar`